server/@types/viewModels/index.d.ts

```diff
@@ -115,6 +115,7 @@ declare module 'viewModels' {
     coursePlannedEndDate?: Date
     isAccredited: boolean
     grade?: string
+    withdrawalReason?: string
```

Comment:
I had to add withdrawal reason to the view model, as it wasn't there previously; hence the changes to the mapper and it's tests below